pyproject.toml

```diff
@@ -0,0 +1,64 @@
+[build-system]
+requires = ["setuptools>=42", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "drydock"
+dynamic = ["version"]
+description = "A Tutor plugin to manage our opinionated Open edX operations"
+readme = "README.md"
+requires-python = ">=3.8"
+license = { text = "AGPLv3" }
+authors = [
+    { name = "eduNEXT" }
+]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: GNU Affero General Public License v3",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+]
+dependencies = [
+    "tutor>=17.0.2,<18.0.0"
```

Comment:
For palm it should be tutor>16, tutor<17

Reply:
Thank you, that was the erro with the workflow